lapis2/src/main/kotlin/org/genspectrum/lapis/config/SequenceFilterFields.kt

```diff
@@ -1,17 +1,28 @@
 package org.genspectrum.lapis.config
 
-typealias FieldName = String
+typealias SequenceFilterFieldName = String
 
-data class SequenceFilterFields(val fields: Map<FieldName, SequenceFilterFieldType>) {
+data class SequenceFilterFields(val fields: Map<SequenceFilterFieldName, SequenceFilterFieldType>) {
     companion object {
         private val nucleotideMutationsField = Pair("nucleotideMutations", SequenceFilterFieldType.MutationsList)
 
-        fun fromDatabaseConfig(databaseConfig: DatabaseConfig) = SequenceFilterFields(
-            fields = databaseConfig.schema.metadata
+        fun fromDatabaseConfig(databaseConfig: DatabaseConfig): SequenceFilterFields {
+            val metadataFields = databaseConfig.schema.metadata
                 .map(::mapToSequenceFilterFields)
                 .flatten()
-                .toMap() + nucleotideMutationsField,
-        )
+                .toMap()
+            val staticFields = listOf(nucleotideMutationsField)
+
+            val featuresFields = if (databaseConfig.schema.features.isNullOrEmpty()) {
+                emptyMap<SequenceFilterFieldName, SequenceFilterFieldType>()
+            } else {
+                databaseConfig.schema.features
+                    .map(::mapToSequenceFilterFieldsFromFeatures)
+                    .toMap()
```

Comment:
"Call chain on collection type may be simplified "
```suggestion
                databaseConfig.schema.features.associate(::mapToSequenceFilterFieldsFromFeatures)
```